<launch>
    <include file="$(find kitti2bag)/launch/replay.launch"/>

    <node pkg="prior_factor_manager" type="prior_factor_manager_node" name="prior_factor_manager_node" output="screen">
        <param name="cov_x" type="double" value="1.0" />
        <param name="cov_y" type="double" value="1.0" />
        <param name="cov_z" type="double" value="100.0" />
        <param name="map_frame" value="gps_reference" />
        <remap from="~/gps" to="/kitti/oxts/gps/fix"/>
        <remap from="~/imu" to="/kitti/oxts/imu"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find prior_factor_manager)/rviz/kitti.rviz" />
</launch>